command-line-local-shortener/main.go

```diff
@@ -88,6 +68,24 @@ func getReq(c *gin.Context) {
 	}
 }
 
+func postReq(c *gin.Context) {
+	longURL := c.PostForm("longURL")
+
+	if longURL == "" {
+		c.String(http.StatusBadRequest, "enter a url to shorten")
+		return
+	} else {
+		result := dataProccess(longURL)
+		if result != "" {
```

Comment:
add a `if err != nil {` check prior to this and move the `else` content into it.

it's better to check for err first than to access value (just a golang standard)